src/automations/checkInactivity.js

```diff
@@ -102,64 +107,72 @@ async function scrapeInactiveIssues(client, references, issues) {
   const iterator = issues[Symbol.iterator]();
 
   for (let issue of iterator) {
-    const inactiveLabel = issue.labels.find(label => {
-      return label.name === client.cfg.activity.inactive;
-    });
-    if (inactiveLabel) continue;
-
-    let time = Date.parse(issue.updated_at);
     const issueNumber = issue.number;
     const repoName = issue.repository.name;
     const repoOwner = issue.repository.owner.login;
     const issueTag = `${repoName}/${issueNumber}`;
 
-    if (time < references.get(issueTag)) time = references.get(issueTag);
+    if (references.get(issueTag).review) {
```

Comment:
Why are you nesting everything? Just `if (!references.get(issueTag).review) continue` should be fine.

Reply:
Thanks, I will immediately make the changes.